src/NServiceBus.RabbitMQ/Configuration/ConnectionStringParser.cs

```diff
@@ -73,6 +73,15 @@ public ConnectionConfiguration Parse(string connectionString)
                 throw new NotSupportedException(message);
             }
 
+            if (ContainsKey("UsePublisherConfirms"))
+            {
+                var message = "The 'UsePublisherConfirms' connection string option has been removed. Use 'EndpointConfiguration.UsePublisherConfirms' instead.";
+
+                Logger.Error(message);
+
+                throw new NotSupportedException(message);
```

Comment:
I'm not sure I agree with this pattern in general, i.e. log an error and then throw an exception. I think we should just throw an exception. However, it matches the other cases so it's fine for the purposes of this PR.